src/modules/mavlink/mavlink_receiver.cpp

```diff
@@ -918,8 +918,10 @@ MavlinkReceiver::handle_message_set_position_target_local_ned(mavlink_message_t
 		/* yaw ignore flag mapps to ignore_attitude */
 		offboard_control_mode.ignore_attitude = (bool)(set_position_target_local_ned.type_mask & 0x400);
 		/* yawrate ignore flag mapps to ignore_bodyrate */
-		offboard_control_mode.ignore_bodyrate = (bool)(set_position_target_local_ned.type_mask & 0x800);
-
+		// offboard_control_mode.ignore_bodyrate = (bool)(set_position_target_local_ned.type_mask & 0x800);
+		offboard_control_mode.ignore_bodyrate_x = (bool)(set_position_target_local_ned.type_mask & 0x800);
```

Comment:
Can we define these masks somewhere?

Reply:
It should really be part of the mavlink spec, but at least for our own sanity we could define them in PX4.